tests/unit/workflows/test_workflows_utils.py

```diff
@@ -524,3 +526,23 @@ def test_grobid_empty_author():
     authors = GrobidAuthors(input_xml)
     assert len(authors) == expected_authors_count
     assert authors.parse_all() == expected_authors
+
+
+def test_get_document_url_for_reference_extraction():
+    file_url = "/api/files/be4ca558-1948-478c-a7f1-05af95b94282/2205.02190.pdf"
+    data = {
+        'documents': [
+            {
+                'key': 'table_of_contents.pdf',
+                'url': file_url
+            },
+            {
+                'key': 'document.pdf',
+            },
+        ],
+    }
+    obj = MockObj(data, {})
+    url = get_document_url_for_reference_extraction(obj)
+    assert current_app.config['SERVER_NAME'] in url
+    assert file_url in url
+    assert 'http' in url
```

Comment:
We have to mock the `current_app`, and we have to update the assert in order to catch errors like the previous I mentioned 😃 

```suggestion
    obj = MockObj(data, {})
    url = get_document_url_for_reference_extraction(obj)
    expected_url = 'http://{0}{1}'.format(server_name, file_url)
    assert expected_url == url
```